ecosystem/manager.py

```diff
@@ -52,6 +52,63 @@ def recompile(self):
         """Recompile `members.json` from human-readable files."""
         self.dao.compile_json()
 
+    def build_website(self):
+        """Generates the ecosystem web page reading `members.json`."""
+        environment = Environment(loader=FileSystemLoader("ecosystem/html/"))
+        website_template = environment.get_template("webpage.html")
+        card_template = environment.get_template("card.html")
+        tag_template = environment.get_template("tag.html")
+        link_template = environment.get_template("link.html")
+
+        sections = {
+            "provider": "",
+            "applications": "",
+            "other": "",
+        }
+
+        projects = DAO("ecosystem/resources/").storage.read()
+
+        count_read_more = 1
+        max_chars_description = 500
+        for _, repo in projects.items():
+            # Card tags
+            tags = ""
+            for label in repo.labels:
+                tags += tag_template.render(color="purple", title=label, text=label)
+
+            # Card links
+            links = link_template.render(url=repo.url, place="repository")
+            if repo.website:
+                links += link_template.render(url=repo.website, place="website")
+
+            # Card description
+            if len(repo.description) - max_chars_description >= 0:
+                description = [repo.description[:400], repo.description[400:]]
```

Comment:
It'd be useful to explain why this is not the same as max_chars_description. Maybe have the 400 number calculated dynamically, like `max_chars_description - 100`. Either way, probably put it as a class level `CONSTANT` with a comment explaining what's going on.

Reply:
I've added another variable called `min_chars_description_hidden` and changed those lines to use another variable called `max_chars_description_visible`, which I think is more readable. Moreover, we avoid calculating any value dynamically.